e2e/testsuite/testconfig.go

```diff
@@ -723,3 +731,18 @@ func modifyClientGenesisAppState(ibcAppState []byte) ([]byte, error) {
 
 	return ibcGenBz, nil
 }
+
+// modifyChannelGenesisAppState takes the existing ibc app state, unmarshals it to a map and removes the `params` entry from ibc channel genesis.
+// It marshals and returns the ibc GenesisState JSON map as bytes.
+func modifyChannelGenesisAppState(ibcAppState []byte) ([]byte, error) {
+	var ibcGenesisMap map[string]interface{}
+	if err := json.Unmarshal(ibcAppState, &ibcGenesisMap); err != nil {
+		return nil, err
+	}
+
+	// be ashamed, be very ashamed
```

Comment:
:heart: 

This is not actually the worst thing ATM, I think we can improve this, but this ended up being very readable and concise compared to the other options we were playing with.

Reply:
yea, agree! let's leave in the comment for comical effect I guess, would be nice to not have to add a new func for every field we need to sanitise, but we can think about that later - string splitting on a `jsonPath` with "." is a bit gross but I'm not sure I have any better ideas atm!